plugins/out_http/http.c

```diff
@@ -342,6 +349,13 @@ void cb_http_flush(void *data, size_t bytes,
                             sizeof(FLB_HTTP_MIME_MSGPACK) -1);
     }
 
+    if (ctx->headertag_name) {
+        flb_http_add_header(c,
+                        ctx->headertag_name,
+                        strlen(ctx->headertag_name),
```

Comment:
Since ctx->headertag_name length never change, can you set it in the context to avoid strlen() every time ?

Reply:
@edsiper Thanks. That makes sense. let me change it and also rebase with current master to make it easier for merge.

Reply:
@edsiper Could you review again? I pre-compute and store in context. THanks.